components/QueryInfo/QueryInfo.tsx

```diff
@@ -0,0 +1,54 @@
+import { CheckCircleIcon, AlertFillIcon } from '@primer/octicons-react'
+import React from 'react'
+import { Spinner } from 'react-bootstrap'
+import styles from '../../scss/queryInfo.module.scss'
+
+type QueryInfo = {
+  data: any
```

Comment:
Can you remove this use of `any`?

Reply:
I'm not super familiar with typescript, but from my knowledgeI think generics should help
https://www.typescriptlang.org/docs/handbook/2/generics.html

Reply:
> I'm not super familiar with typescripbut from my knowledge, I think generics should help https://www.typescriptlang.org/docs/handbook/2/generics.html

I wasn't sure how the generic type for a prop work, so I googled a bit and ended up with what's in the code. According to the code examples I've seen and based on my understanding, `T` stands for the prop type. I'm not certain though, there isn't a clear explanation of what `T` is.